doc/introduction/operations.rst

```diff
@@ -49,33 +52,27 @@ Qubit gates
 .. autosummary::
     :nosignatures:
 
+    ~pennylane.CNOT
```

Comment:
@AroosaIjaz could you change the ordering back here to match the original? Thanks!

Reply:
I will move S, T with single qubit non-parametrized gates and CZ with multiple qubit non-parametrized gates